Models/Employee.cs

```diff
@@ -237,6 +237,13 @@ EmployeeInfoLookupService infoLookupService
         // field. This should only be run when the Employee is created.
         public void InstantiateFields()
         {
+            if (Address1 == null) {
+                Address1 = "TEMPORARY"; // TODO: DEFINITELY REMOVE
```

Comment:
Is there an issue or notes on your side to remove this?

Reply:
Created a ticket: https://github.com/bcgov/ExitSurveyAdmin/issues/209